src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/stats/eval/impl/Count.java

```diff
@@ -0,0 +1,38 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.stats.eval.impl;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.stats.eval.Statistics;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.stats.eval.impl.vals.AggregateValue;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class Count implements StatisticImpl<AggregateValue> {
```

Comment:
StatisticImpl definition seems to be missing. Also, we can just call it Statistic.

Reply:
Can you elaborate on the definition seems to be missing part ? There is already an enum called Statistics which contains all the aggregate types.

Reply:
I can find the class now. I was not able to find the class in the earlier commits.